R/dependencies.R

```diff
@@ -698,5 +704,32 @@ get_local_pkgs_from_config <- function() {
 
 
 
+#' TODO roxygen
+#' @export
+update_stage_dep_yaml_with_direct_deps <- function(project = ".", feature = NULL,
```

Comment:
name too long and difficult to read, inconsistent with above
maybe `update_stageddeps_yaml_with_direct_deps`